dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/CommandLine.java

```diff
@@ -47,9 +48,10 @@ public static void main(String[] args)
             System.exit(1);
         Class<?> runClass = loadCommand(allClasses, ns.get(Constants.COMMAND));
         DockerfileGitHubUtil dockerfileGitHubUtil = initializeDockerfileGithubUtil(ns.get(Constants.GIT_API));
+        DockerfileS3Util dockerfileS3Util = initializeDockerfileS3Util();
```

Comment:
No need to initialize both. We shouldn't need to create a "util" for S3 and keep all methods with the S3Store you created below.